### PR TITLE
Feat/about

### PR DESCRIPTION
**Pull request by spiderCSman**
rowdyhacks favicon and navbar logo

### CODE REVIEW

**Comment by christianhelp** [comment]
Btw, this isn't how you assign reviewers. Look to the top right corner and you should see a "reviewers" option.

File: apps/web/src/app/favicon.ico
Comment (christianhelp) [inline apps/web/src/app/favicon.ico:1]: The favicon should not be this large. I do not think this was created properly.

File: packages/config/hackkit.config.ts [hunk 1]
```diff
@@ -861,10 +861,10 @@ const c = {
 		guide: "https://go.rowdyhacks.org/discord",
 	},
 	icon: {
-		sm: "/img/logo/hackkit.svg",
-		md: "/img/logo/hackkit-md.png",
-		lg: "/img/logo/hackkit-lg.png",
-		svg: "/img/logo/hackkit.svg",
+		sm: "/img/logo/rhbttf.svg",
```
Comment (christianhelp) [inline packages/config/hackkit.config.ts:864]: question: did you check to see if all of these files exist where you say they do?

**Comment by christianhelp** [comment]
Question: this branch claims it is referring to the `about` section, but the files changed and the description show no changes to the about section.

File: packages/config/hackkit.config.ts (hunk 1 above)
Comment (spiderCSman) [reply packages/config/hackkit.config.ts:864]: yes they do

File: apps/web/src/app/favicon.ico
Comment (spiderCSman) [reply apps/web/src/app/favicon.ico:1]: how about now?

Comment (christianhelp) [reply apps/web/src/app/favicon.ico:1]: Hm seems to say an error occurred while trying to load it.

Comment (spiderCSman) [reply apps/web/src/app/favicon.ico:1]: i created a new pull and on that one i import the favicon after taking the svg and putting it in photoshop and converting to favicon

File: packages/config/hackkit.config.ts (hunk 1 above)
Comment (spiderCSman) [reply packages/config/hackkit.config.ts:864]: the ones on lines 865 and 866 do not exist

Comment (spiderCSman) [reply packages/config/hackkit.config.ts:864]: should i request the design team to make them?

**Pull request closed without merging**